### ISSUE TITLE
-IncludeExperimental is not implemented in .\build.ps1

### ISSUE THREAD

**Issue by jeanfrancoislarente** [Sitecore/docker-images#202]
Nice change to have the build.ps1 support so many options. One option that is missing which would be nice is the `-ExperimentalTagBehavior` flag.

```
[Parameter(Mandatory = $false)]
[ValidateSet("Include", "Skip")]
[string]$ExperimentalTagBehavior = "Skip"
```

Do you have a couple of moments to add that @michaellwest  ?

**Comment by michaellwest** [comment]
I can add it. Expect a PR this afternoon.

**Comment by jeanfrancoislarente** [comment]
You da man!

**Comment by kkum** [comment]
If this is needed for specific tags wouldn't it be better to have it set when adding tags to the list
```    $ExperimentalTagBehavior = "Skip"
    foreach ($scv in $SitecoreVersion)
    {
        $tags.Add("sitecore-assets:$($scv)-nanoserver-$($windowsVersionMapping[$wv])") > $null
        if ([Version]$scv -ge [Version]"9.3.0" -and [int]$windowsVersionMapping[$wv] -gt 1809)
        {
            $ExperimentalTagBehavior = "Include"
            $tags.Add("sitecore-ps:$($scv)-nanoserver-$($windowsVersionMapping[$wv])") > $null
        }```

**Comment by michaellwest** [comment]
The experimental behavior is all-or-none based on the current `Invoke-*` commands. If you only want certain versions to be experimental then call build.ps1 multiple times.

**Comment by michaellwest** [comment]
I took another look and I can see how with the changes introduced by #194, you could have experimental included without even knowing it. I'll see what changes I can make to account for this.

**Comment by michaellwest** [comment]
@kkum After some good head scratching I think I have this solved. Image names are now much more dynamic due to #200 .

**Comment by jeanfrancoislarente** [comment]
resolved by #200
